ember.js

```diff
@@ -12,11 +12,14 @@ module.exports = {
       '_super',
       '__container__',
     ]}],
-    'consistent-return': 0,
+    'consistent-return': 1,
```

Comment:
I find this rule annoying because it dislikes early returns

Reply:
That's why it's a warning.